docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Documentation Syncing with GitHub Actions
```

Comment:
I'd drop "Documentation"

Reply:
ah I understand now :) I think the title `Sync of Documentation through Github Actions` (or something similar) would be less prone to misunderstanding, because at first I thought that this just says "Documentation (of) syncing with Github Actions"